test/end-to-end/offer.js

```diff
@@ -2004,6 +2040,58 @@ describe("Offer endpoint tests", () => {
             });
         });
 
+        describe("Same 'publishDate' and 'publishEndDate'", () => {
+
+            const dateNow = (new Date(Date.now() + (DAY_TO_MS * 5))).toISOString();
+            const dateAfter = (new Date(Date.now() + (DAY_TO_MS * 10))).toISOString();
+            let offer;
+
+            beforeAll(async () => {
+
+                offer = await Offer.create({
```

Comment:
delete previous offers, and create this one using generateTestOffer util

Reply:
In this situation, deleting offers causes tests that happen after this one to have erroneous behavior (related to #169 ). Maybe i should keep the `Offer.deleteMany({});` line but comment it out.